site/docs/numbers/manageInventory.mdx

```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
+                </TelephoneNumbers>
+                <SubscriberInformation>Joes Garage</SubscriberInformation>
+                <UseType>RESIDENTIAL</UseType>
+                <Visibility>PUBLIC</Visibility>
+            </LidbTnGroup>
+            <LidbTnGroup>
+                <TelephoneNumbers>
+                    <TelephoneNumber>8888595935</TelephoneNumber>
+                </TelephoneNumbers>
+                <SubscriberInformation>Joes Storage</SubscriberInformation>
+                <UseType>RESIDENTIAL</UseType>
+                <Visibility>PRIVATE</Visibility>
+            </LidbTnGroup>
+        </LidbTnGroups>
+    </LidbOrder>'
+```
+
+</TabItem>
+</Tabs>
+
+> Response
+
+```xml
+<LidbOrderResponse>
+    <LidbOrder>
+        <orderId>1d863522-c9bf-430c-8e79-cc63cad04a1d</orderId>
+        <CustomerOrderId>[String]</CustomerOrderId>
+        <ProcessingStatus>RECEIVED</ProcessingStatus>
+        <ErrorList>
+            <Error>
+                <TelephoneNumber>8888595935</TelephoneNumber>
+                <Code>11020</Code>
+                <Description>Cannot add LIDB to a telephone number that already has LIDB or is in process</Description>
+            </Error>
+        </ErrorList>
+        <LidbTnGroups>
+            <LidbTnGroup>
+                <TelephoneNumbers>
+                    <TelephoneNumber>8045030097</TelephoneNumber>
+                    <TelephoneNumber>8045030098</TelephoneNumber>
+                </TelephoneNumbers>
+                <SubscriberInformation>Joes Garage</SubscriberInformation>
+                <UseType>RESIDENTIAL</UseType>
+                <Visibility>PUBLIC</Visibility>
+            </LidbTnGroup>
+            <LidbTnGroup>
+                <TelephoneNumbers>
+                    <TelephoneNumber>8888595935</TelephoneNumber>
+                </TelephoneNumbers>
+                <SubscriberInformation>Joes Storage</SubscriberInformation>
+                <UseType>RESIDENTIAL</UseType>
+                <Visibility>PRIVATE</Visibility>
+            </LidbTnGroup>
+        </LidbTnGroups>
+    </LidbOrder>
+</LidbOrderResponse>
+```
+
+</TabItem>
+
+<TabItem value="dlda">
+
+The DLDA service allows the association of a name and address listing with a Telephone Number or Telephone Numbers in the Bandwidth Phone Number Phone Number system.  The DLDA API follows the asynchronous order processing request model described elsewhere in this document, where a request is submitted, an order ID is returned as an ID of that request or order, and then the request is fulfilled in the background.  The status of the request is reflected in the status of the order, and can be queried at any time.
+
+To create a DLDA Order, make a <Highlight color="#079CEE">POST</Highlight> request to our [Create DLDA Order API](../../../../apis/numbers#tag/accounts/operation/CreateDldaOrder) endpoint.
+This can be done through tools like [Postman](https://github.com/Bandwidth/postman) or in the command line.
```

Comment:
Change "in the command line" to "cURL"

Reply:
@gocab-bw I have been using this sentnce in previeus guids also. For example in `seach and order numbers`. Should I change it then?

Reply:
Yes, please change in other places.

Reply:
I've already changed them also